src/testing/cluster.zig

```diff
@@ -486,7 +486,7 @@ pub fn ClusterType(comptime StateMachineType: fn (comptime Storage: type, compti
                 .message_sent => |message| {
                     cluster.state_checker.on_message(message);
                 },
-                .commit => {
+                .committed => {
```

Comment:
Tangentially related, but one cute naming convention for events is `onDidDoSomething/onWillDoSomething` from VS Code (https://code.visualstudio.com/api/references/vscode-api), eg `onDidSaveTextDocument/onWillSaveTextDocument`. They name all events that way, to remove ambiguity about when, exactly, the hoot is called relative to the event.